### PR TITLE
more fixes

### PR DESCRIPTION
**Pull request by redonkulus**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,107 +0,0 @@
-# Welcome
-
-These docs are meant to help you get started building isomorphic apps with the
-Fluxible libraries.
-
-
-## Prerequisites
```
Comment (Vijar) [inline docs/getting-started.md]: I feel like we should hold on to this section somewhere

Comment (Vijar) [inline docs/getting-started.md]: I could be convinced otherwise as well

Comment (jedireza) [inline docs/getting-started.md]: Hold on to prerequisites?

**Comment by jedireza** [comment]
:+1:

File: docs/getting-started.md (hunk 1 above)
Comment (redonkulus) [inline docs/getting-started.md]: I think we could be explicit about flux and react but node and express I don't think we have to mention.

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Quick Start
+
+Do stuff
```
Comment (Vijar) [inline docs/quick-start.md]: How about:

+If you wan't to jump right into it, check out our examples:
+
+ - [Chat](https://github.com/yahoo/flux-examples/tree/master/chat) - Port of [Facebook's Flux chat example](https://github.com/facebook/flux/tree/master/examples/
+ - [Routing](https://github.com/yahoo/flux-examples/tree/master/routing) - Simple isomorphic routing in flux flow
+ - [To Do](https://github.com/yahoo/flux-examples/tree/master/todo) - Port of [ToDo MVC](https://github.com/tastejs/todomvc)

**Pull request merged**